pkg/game/final_scoring_test.go

```diff
@@ -0,0 +1,633 @@
+package game
+
+import (
+	"testing"
+
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/deck"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/elements"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/game/position"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/stack"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/feature"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/side"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tiles/tiletemplates"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+/*
+  All tests contains the board visualization. Each tile is represented by 5x5 ascii characters
```

Comment:
I looked up the format's description from the #52 PR (`pkg/end_tests/four_player_game_test/four_player_game_test.go`) but I want to note that it doesn't explain how `[`/`]` characters should be interpreted. They are related to monasteries but I'm not entirely sure what their placement means since I've seen variants where they occupy 3 rows and a case where it only occupies one row. Not a blocker for this PR but I would want some clarifications added in #52.

Reply:
I see in main board drawing in four_player_game_test I accidentaly drew monastery using single brackets[ ] insted of 3 pairs, but further in the test was drawn correctly. If you want I can change it to cross or another shape. I needed to somehow give it some surface to visualize that there is a meeple in/on it. That's why it not supposed to be drawn with single bracket pair

Reply:
> I see in main board drawing in four_player_game_test I accidentaly drew monastery using single brackets[ ] insted of 3 pairs, but further in the test was drawn correctly.

Ah, so it was just an unintentional inconsistency, that's fine - just make sure to fix that in #52 :) My comment about describing this in #52 still stands though ofc that's irrelevant to this PR.

> If you want I can change it to cross or another shape. I needed to somehow give it some surface to visualize that there is a meeple in/on it.

I'm mostly indifferent about the format right now since I have trouble visualizing the scenarios using these ASCII formats without recreating them in a fully graphical form anyway. I just noticed an inconsistency so wanted to know, if there's more to it. There are probably some things that could be done to make the ASCII format more readable but I don't know which part of it would need changing. I'd hold off on changing anything with it until this PR is merged, perhaps even until *after* #63 is merged.